document-legacy/src/layers/layer_info.rs

```diff
@@ -321,15 +324,17 @@ impl Layer {
 					transforms.pop();
 					self.cache.clear();
 					self.cache_dirty = true;
-					return "";
+					return ("", false);
 				}
 			}
 		}
 
+		let mut success = true;
```

Comment:
Here also, just the word "success" doesn't clearly indicate the precise meaning here. It's not obvious to me what it means for the render to be successful, or what the implications are in either case.